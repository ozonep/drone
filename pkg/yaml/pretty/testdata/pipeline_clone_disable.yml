---
kind: pipeline
name: default

clone:
  disable: true

steps:
- name: webhook
  image: alpine
  commands:
  - curl -x POST http://abc.com:80/~smith/home.html
