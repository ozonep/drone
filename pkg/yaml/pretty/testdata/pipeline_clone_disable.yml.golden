---
kind: pipeline
name: default

platform:
  os: linux
  arch: amd64

clone:
  disable: true

steps:
- name: webhook
  image: alpine
  commands:
  - curl -x POST http://abc.com:80/~smith/home.html

...
